openedx/features/discounts/tests/test_views.py

```diff
@@ -46,7 +46,7 @@ def test_course_user_discount(self):
         assert expected_payload['discount_applicable'] == response.data['discount_applicable']
 
         # make sure that the response matches the expected response
-        response_payload = jwt.decode(response.data['jwt'], verify=False)
+        response_payload = jwt.decode(response.data['jwt'], algorithms=["HS256"], options={"verify_signature": False})
```

Comment:
https://pyjwt.readthedocs.io/en/stable/changelog.html#dropped-deprecated-verify-param-in-jwt-decode